revocation_list.go

```diff
@@ -39,9 +41,19 @@ func (r RevocationList) ClearRevocation(pubKey string) {
 }
 
 // IsRevoked checks if the public key is in the revoked list with a timestamp later than
-// the one passed in. Generally this method is called with time.Now() but other time's can
+// the one passed in. Generally this method is called with an issue time but other time's can
```

Comment:
Sorry, I just merged #113 that was a port of some changes that were made in v2/ but not v1/ but were done in the branch_v1 :-). So you should rebase and see if there are some changes that are no longer needed.